### DISCUSSION TITLE
QOL Feature. Auto save

### DISCUSSION THREAD

**Discussion by whitevamp** [Rem0o/FanControl.Releases#594] [Ideas]
would love to see a auto save feature added. IE: after you have made a change to a sensor it will auto save.
or  I would even settle for a notification of " you have unsaved settings are you sure you want to exit. " when you go to close the APP..

**Comment by Rem0o** [comment]
Figured out when you press exit from the tray icon, it doesn't ask for save.
Every other way to exit the app it does ask. Blind spot.

Will fix.

**Comment by AntaloFis** [comment]
Sorry for reviving an old thread but it would be really great to have an auto save feature as an option (if not by default). I've had now many times where I've lost configuration changes after long stress tests and tweaking because I didn't remember to save them or the computer went to sleep and they were lost when waking up, or the power went off and same result. Last one this morning and it's been frustrating especially because I really like this software and this makes me want to stop using it. Would this be hard to implement? Thank you very much.
